lib/uswds/rails/engine.rb

```diff
@@ -0,0 +1,18 @@
+module Uswds
+  module Rails
+    class Engine < ::Rails::Engine
+      initializer 'uswds-rails.assets.precompile' do |app|
+        %w(stylesheets javascripts fonts images).each do |sub|
+          app.config.assets.paths << root.join('assets', sub).to_s
+        end
+
+        # sprockets-rails 3 tracks down the calls to `font_path` and `image_path`
+        # and automatically precompiles the referenced assets.
+        unless Sprockets::Rails::VERSION.split('.', 2)[0].to_i >= 3
+          #app.config.assets.precompile << %r(bootstrap/glyphicons-halflings-regular\.(?:eot|svg|ttf|woff2?)$)
+          # TODO do we need this for uswds?
```

Comment:
looked at commit history on bootstrap-sass and looks like this is related to anything before sprockets-rails 3 https://github.com/twbs/bootstrap-sass/commit/3e0fafa09a362e601372d078926398c59ba91d67#diff-db4fa8404e2c8c4f78edc5fcf857dd4d

since that was released almost a year ago, I would be OK with not including this and instead including a comment about this gem only being compatible sprockets-rails v 3+ 

https://rubygems.org/gems/sprockets-rails/versions